bin.rams/Makefile

```diff
@@ -122,13 +122,11 @@ OBJ = $(ARC)($(MODEL)/lib/an_header.o) \
       $(ARC)($(MODEL)/mpi/mpass_init.o) \
       $(ARC)($(MODEL)/mpi/mpass_lbc.o)  \
       $(ARC)($(MODEL)/mpi/mpass_nest.o) \
```

Comment:
These are the same two source files removed below.

Reply:
Notably, write_scm isn't deleted from the Makefile despite being deleted below. Did I make a mistake in copying here or should the write_scm line be deleted from the makefile? @smsaleeb

Reply:
Never mind, GitHub just didn't recognize this as a moved file. Apologies.